Automattic/jetpack
# My Jetpack: add loading animation to recommendations step

## Proposed changes:

* This PR adds a quick loading animation when fetching recommendations that are suitable for your site. This happens after a site is connected and you fill out the recommendations survey.


https://github.com/user-attachments/assets/009ee93e-bf0b-4043-83df-73c701aa1738


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Spin up a JN site with Jetpack Beta pointing to this branch
* Go to My Jetpack
* Connect your site, confirm the animation looks like the one above
* Answer the survey, submit your response and confirm the animation looks similar to the one in the vide
